trojsten/polls/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+
+from .models import Answer, Question, Vote
+
+admin.site.register(Question)
+admin.site.register(Answer)
+admin.site.register(Vote)

```

Comment:
Chceme teda vobec mat vote v adminovi?

Reply:
Toto pridava Vote na adminsku stranku?
Potom ho tam nechceme, ale stale chceme, aby ho videli superadmini. Ako sa to spravi?

Reply:
Ked to zmazeme, tak to nebudu vidiet ani superadmini. Co je asi to co koniec koncov chceme, ci?

Ak chceme aby to videli superadmini, tak to tu mozme nechat.

Reply:
Dobre, zmazem to teda.